SPECS/java-cup/java-cup.spec

```diff
@@ -1,65 +1,48 @@
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
-#
-# spec file for package java-cup
-#
-# Copyright (c) 2019 SUSE LINUX GmbH, Nuernberg, Germany.
-#
-# All modifications and additions to the file contributed by third parties
-# remain the property of their copyright owners, unless otherwise agreed
-# upon. The license for this file, and modifications and additions to the
-# file, is the same license as for the pristine package itself (unless the
-# license for the pristine package is not an Open Source License, in which
-# case the license is the MIT License). An "Open Source License" is a
-# license that conforms to the Open Source Definition (Version 1.9)
-# published by the Open Source Initiative.
-
-# Please submit bugfixes or comments via https://bugs.opensuse.org/
-#
-
-
 %define with_bootstrap 1
-%define cvs_version        11a
-%define real_name   java-cup
-%bcond_without                bootstrap
+%define cvs_version    11b
+%define real_name      java-cup
+%define pub_date       20160615
+%bcond_without         bootstrap
+
+Summary:        LALR Parser Generator in Java
 Name:           java-cup
 Version:        0.11
-Release:        30%{?dist}
-Summary:        LALR Parser Generator in Java
+Release:        31%{?dist}
 License:        HPND
 Group:          Development/Libraries/Java
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
 Url:            http://www2.cs.tum.edu/projects/cup/
-# https://www2.in.tum.de/WebSVN/dl.php?repname=CUP&path=/develop/&rev=0&isdir=1
-Source0:        develop.tar.bz2
+# http://www2.cs.tum.edu/projects/cup/releases/%{name}-src-%{cvs_version}-%{pub_date}.tar.gz
+Source0:        http://www2.cs.tum.edu/projects/cup/releases/%{name}-src-%{cvs_version}-%{pub_date}.tar.gz#/%{name}-%{version}b.tar.gz
 Source1:        java-cup.script
 Source2:        java-cup-generated-files.tar.bz2
 # From          http://www2.cs.tum.edu/projects/cup/
 Source3:        java-cup.license
 Patch1:         java-cup-no-classpath-in-manifest.patch
-Patch2:         java-cup-no-cup-no-jflex.patch
-Patch3:         java-cup-classpath.patch
-# Missing symbolFactory initialization in lr_parser, causes sinjdoc to crash
-Patch4:         java-cup-lr_parser-constructor.patch
```

Comment:
I think we forgot to remove this one and it's still in the `SPECS/java-cup` folder.

Reply:
good catch!